.github/workflows/my-ci-pipeline.yml

```diff
@@ -63,16 +63,12 @@ jobs:
            
           
   deploy:
-    needs: test
+    needs: build
     if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main'}}
-    name: Deploy to Heroku
+    name: Deploy to Azure
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
-      - uses: gonuit/heroku-docker-deploy@v1.3.3
-        with:
-          email: philw10@yahoo.co.uk         
-          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
-          heroku_app_name: phil-todo-app
-          docker_options: "--target production"
+      - name: Trigger webhook
+        run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK }}
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results